rllib/policy/policy.py

```diff
@@ -1389,17 +1344,13 @@ def _initialize_loss_from_dummy_batch(
             sample_batch_size
         )
         self._lazy_tensor_dict(self._dummy_batch)
-        # With RL Modules you want the explore flag to be True for initialization
-        # of the tensors and placeholder you'd need for training.
-        explore = self.config.get("enable_rl_module_and_learner", False)
-
+        explore = False
         actions, state_outs, extra_outs = self.compute_actions_from_input_dict(
             self._dummy_batch, explore=explore
         )
-        if not self.config.get("enable_rl_module_and_learner", False):
-            for key, view_req in self.view_requirements.items():
-                if key not in self._dummy_batch.accessed_keys:
-                    view_req.used_for_compute_actions = False
+        for key, view_req in self.view_requirements.items():
```

Comment:
We should be able to remove now also the `view_requirements` from the `Catalog`, shouldn't we? In addition remove all the `view_requirement`  methods in the RLModule

Reply:
done